oaknational/Oak-Web-Application
# refactor(storybook): tidy up stories, hopefully fix broken ones in prod

## Description

- Tidy stories to make them consistent (remove unnecessary `div` wrappers, common names)
- Hopefully this might fix the errors we see on some components in the deployed versions

## How to test

1. Go to {cloud link}
2. Click on "Lesson Header
3. You should see the component (not a "displayName" of undefined error)
4.
